### PR TITLE
Backport: Use `gh pr create` for the final push to GitHub

### PR DESCRIPTION
**Pull request by zerok**
This will already fill out all the necessary fields to make the process for the developer easier. The instructions for the manual backport will now look like this:

```bash
# Fetch latest updates from GitHub
git fetch
# Create a new branch
git switch --create backport-16-to-v10.0.x origin/v10.0.x
# Cherry-pick the merged commit of this pull request and resolve the conflicts
git cherry-pick -x 5bd3e5107a1ab75b8723916b0bc3c5206861afde
# When the conflicts are resolved, stage and commit the changes
git add . && git cherry-pick --continue
# If you have the GitHub CLI installed: Push the branch to GitHub and a PR:
gh pr create --title "[v10.0.x] Update README.md" --body "Backport 5bd3e5107a1ab75b8723916b0bc3c5206861afde from #16" --label backport --base v10.0.x --milestone 10.0.x --web
# If you don't have the GitHub CLI installed: Push the branch to GitHub and manually create a PR:
git push --set-upstream origin backport-16-to-v10.0.x
# Remove the local backport branch
git switch main
git branch -D backport-16-to-v10.0.x
```

### CODE REVIEW

File: backport/backport.ts [hunk 1]
```diff
@@ -180,17 +180,21 @@ const backportOnce = async ({
...
+	title,
 }: {
 	base: string
 	commitToBackport: string
 	errorMessage: string
 	head: string
+	title: string
 }) => {
+	const backportMilestone = base.startsWith('v') ? base.substring(1) : base
+	const escapedTitle = title.replaceAll('"', '\\"')
```
Comment (dsotirakis) [inline backport/backport.ts:197]: We need to add the branch name to the title as well `[vX.Y.Z]`

Comment (dsotirakis) [reply backport/backport.ts:197]: Doesn't happen for failing PRs I think :/

Comment (zerok) [reply backport/backport.ts:197]: Hm... the title is templated in https://github.com/grafana/grafana-github-actions/pull/150/files#diff-4ba6a76ffc49abf7b860d7951507573e42bc5a66d7fe0480297b287ad8c2fdc8R342 and the call to `getFailedBackportCommentBody` happens after that so I think this should work 🙂

Comment (zerok) [reply backport/backport.ts:199]: Tested on a private repo with settings similar to what we have in grafana/grafana. I've added the generated output to the description, so it seems to work :)

**Review by dsotirakis — APPROVED** [review]

**Pull request merged**